### ISSUE TITLE
Can u consider support lerobot dataset training?

### ISSUE THREAD

**Issue by lucasjinreal** [declare-lab/nora#9]
As many dataset are using lerobot format now

**Comment by hungchiayu1** [comment]
Thanks for the suggestion! We will try and support lerobot dataset training

**Comment by lucasjinreal** [comment]
thanks for the helpful information, looking forward to lerobot dataset support and more simulation data support

**Comment by xliu0105** [comment]
It is very easy to convert the existing nora to be trained with the lerobot dataset, and I have already done so.

**Comment by hungchiayu1** [comment]
@xliu0105 Did you notice any performance drop when finetuning with lerobot dataset? For me, when I perform libero finetuning using lerobot dataset, I notice a ~10% drop in success rate for libero evaluation. As of now I m not too sure with the reason behind this. One of the possible reason I could think of is that Nora's dataloader maps grippler action as (0 = close, 1 = open). While Libero dataset in lerobot format has (-1 = open, 1 = close).

**Comment by lucasjinreal** [comment]
this is why I ask author support, we also had already supported lerobot training, but seems there are some miss alignment between them. Am hoping the author release a official version of it.

**Comment by hungchiayu1** [comment]
@xliu0105 Thanks! Looking forward to hear your feedback! During pretraining of Nora, the loss converge to about ~1.1.

**Comment by hungchiayu1** [comment]
I was able to replicate libero object finetuning in Lerobot dataset format after mapping grippler action to (0 = close, 1 = open). Will continue to run experiment for other libero task.

**Comment by hungchiayu1** [comment]
@lucasjinreal Sure. I will release it very soon

**Comment by xliu0105** [comment]
@hungchiayu1 Yes, my dataset has action_dim=16, but will this cause such a huge performance degradation?

If I want to train nora from scratch, do I just need to load the original qwen-2.5-VL model and modify the dimension of the embedding layer?
